### PR TITLE
documentation/ZENKO-1721_metrics_for_transition_policies

### PR DESCRIPTION
**Pull request by wabernat**
This PR:
Adds a reference section for Prometheus access to Lifecycle Metrics

Adds a  "howto" section in operation for accessing Prometheus dashboard.

This fixes #ZENKO-1721 (again)

I am PTO Thursday and Friday. Whatever you need to do to make this go through, please do it.

### CODE REVIEW

File: docs/docsource/reference/prometheus/index.rst [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+
+You can use either the Prometheus API or the Prometheus UI and the PromQL 
+query language to access these metrics.
+
+Below is an few example. See the Prometheus documentation for a full PromQL
+reference.
+
+  - Compute the total average throughput, in bytes per second, of
+    successful replications triggered by lifecycle transitions over
+    the last 5 minutes::
```
Comment (bennettbuchanan) [inline docs/docsource/reference/prometheus/index.rst:144]: ~minutes::~  minutes:

**Review by bennettbuchanan — APPROVED** [review]

File: docs/docsource/reference/prometheus/index.rst [hunk 2]
```diff
@@ -0,0 +1,146 @@
+Prometheus
+==========
+
+One of several dashboards available to Zenko users, Prometheus provides insight
+into Kubernetes functionalities not accessible through the Backbeat API.
```
Comment (jonathan-gramain) [inline docs/docsource/reference/prometheus/index.rst]: I would rephrase to:

> One of several dashboards available to Zenko users, Prometheus provides insight into Kubernetes platform (metalk8s) and Zenko functionality.

Because Prometheus scope is larger than what Backbeat API is meant to expose, as it covers Cloudserver, MongoDB or other platform-wide components not related to backbeat, I don't think it's worth mentioning Backbeat API here.

What is mentioned below in "Exposure through the Backbeat API is under development" should be enough I think.

File: docs/docsource/reference/prometheus/index.rst [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+
+  - **contentLengthRange:** This range separates the metric values in different
+    content-length (object size) buckets to offer more meaningful elapsed 
+    time metrics grouped by object size ranges. The labels used are: “<10KB”,
+    “10..30KB”, 30..100KB”, […], “300GB..1TB”, “>1TB”.
+
+Kafka Metrics
+~~~~~~~~~~~~~
+
+Kafka provides its own set of Prometheus metrics, one of which is
```
Comment (jonathan-gramain) [inline docs/docsource/reference/prometheus/index.rst]: I'm thinking other Kafka metrics could also be useful, though I did not take time to go through all of them and find use cases for lifecycle monitoring. So the one I'm presenting here "kafka_consumergroup_lag" is definitely useful, there are probably others. Now I let you rephrase this appropriately :wink:

File: docs/docsource/reference/prometheus/index.rst [hunk 4]
```diff
@@ -0,0 +1,146 @@
...
+    - backbeat-lifecycle-object-processor-group: The unique consumer
+      group of the “backbeat-lifecycle-object-tasks” topic.
+
+    - backbeat-gc-consumer-group: The unique consumer group of the
+      “backbeat-gc” topic
+
+You can use either the Prometheus API or the Prometheus UI and the PromQL 
+query language to access these metrics.
+
+Below is an few example. See the Prometheus documentation for a full PromQL
```
Comment (jonathan-gramain) [inline docs/docsource/reference/prometheus/index.rst]: As discussed offline, you might say for now "Below is one example".

File: docs/docsource/reference/prometheus/index.rst (hunk 1 above)
Comment (wabernat) [reply docs/docsource/reference/prometheus/index.rst:144]: No... This is an artifact of RST. The double-colon sets up the code section that follows.

File: docs/docsource/reference/prometheus/index.rst (hunk 2 above)
Comment (wabernat) [reply docs/docsource/reference/prometheus/index.rst]: done

File: docs/docsource/reference/prometheus/index.rst (hunk 4 above)
Comment (wabernat) [reply docs/docsource/reference/prometheus/index.rst]: Your diplomatic papers are on their way, sir!

File: docs/docsource/reference/prometheus/index.rst (hunk 3 above)
Comment (wabernat) [reply docs/docsource/reference/prometheus/index.rst]: Added a catch-all sentence at the end of the section.

**Comment by wabernat** [comment]
@bert-e approve

File: docs/docsource/reference/prometheus/index.rst (hunk 1 above)
Comment (bennettbuchanan) [reply docs/docsource/reference/prometheus/index.rst:144]: I see. It appears my RST parsing skills are lacking. 😆

**Comment by jonathan-gramain** [comment]
I corrected a minor inaccuracy in the description of "contentLengthRange" label.

**Review by jonathan-gramain — APPROVED** [review]

**Review by ssalaues — APPROVED** [review]
